### DISCUSSION TITLE
F1 game telemetry on "marshall board"

### DISCUSSION THREAD

**Discussion by theissondergaard** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#266] [Show and tell]
Thanks for a great library!! Used it to build a Marshall Board next to my racing sim, showing live data from the F1 racing game, by parsing the telemetry data. A fun project with more features added as we come up with them :)

<img width="348" alt="Screenshot 2022-03-15 at 16 05 20" src="https://user-images.githubusercontent.com/6554389/158408001-589d169d-1755-497f-8f2c-d162d8fdc2e5.png">

https://youtu.be/bAyUthLpZM8

(inspired by real life: https://www.emmotorsport.com/solutions/t1)

**Comment by DarrylStrong** [comment]
It **is** a great library, It has made us move from using an expensive PIC32 based CPU board to ESP32 boards for most of our products. 

We designs and manufacture electronic displays and our flagship product is Captioning and Surtitling live shows. For this we use a range of signs which are mostly 384x64 pixels, P2.5 to P5 pitch. (www.sopratext.com) (https://homepages.plus.net/signblazer/SopraText/)

I have also used the same hardware to make a rail departure board. (https://youtu.be/H7iUEQLvT1s)

We designed a small CPU board which sits on the back of the HUB75 connector for super easy use.

Looking for more projects to use this library in.

Cheers!
